pkg/util/generate/generate.go

```diff
@@ -15,6 +16,27 @@ func KeyFileContents() (string, error) {
 	return generateRandomString(500)
 }
 
+// RandomFixedLengthLowercaseAlphanumericString generates a random fixed-length string with characters in a certain range.
+func RandomFixedLengthLowercaseAlphanumericString(n int) (string, error) {
```

Comment:
this function name isn't strictly true, this actually creates a  valid `DNS1123Label`, maybe `RandomValidDNS1123Label`?

Reply:
Done! Renamed to `RandomValidDNS1123Label`.